app/views/admin/creator_settings/new.html.erb

```diff
@@ -41,6 +51,11 @@
   <%= render "form" %>
   <div class="crayons-field mt-6 align-left">
     <%= submit_tag "Finish", class: "crayons-btn btn--primary" %>
+    <% if @help_url %>
+      <a class="admin-help-button" href="<%= @help_url %>" target="_blank" rel="noopener noreferer">
+        <%= inline_svg_tag("circle-question.svg", aria: true, title: "Forem Admin Guide", class: "crayons-btn crayons-btn--icon-rounded crayons-btn--s") %>
+      </a>
+    <% end %>
```

Comment:
I'm having a difficult time lining the "help" icon up with the finish button as shown in this design mockup, though I'm still working on it:
![](https://user-images.githubusercontent.com/108287/116532347-6d033700-a8e0-11eb-8dae-88bf6164f682.png)